client/state/selectors/get-rewind-state.js

```diff
@@ -0,0 +1,18 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import { get } from 'lodash';
+
+/**
+ * Get the entire Rewind state object.
+ *
+ * @param {Object} state Global state tree
+ * @param {number|string} siteId the site ID
+ * @return {Object} Rewind state object
+ */
+export default function getRewindState( state, siteId ) {
+	return get( state.rewind, siteId, {} );
```

Comment:
we probably shouldn't return `{}` here because that would be an invalid state. if we need to we can introduce (from the diagram in the p2) some kind of `uninitialized` state instead